Judge/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/service/JudgeQueueListener.java

```diff
@@ -23,18 +23,22 @@ public JudgeQueueListener(QueueService queueService, LoggerService logger) {
     }
 
     @RabbitListener(queues = "JudgePlayerShootResultQueue")
-    public void listenOnJudgePlayerShootResultQueue (PlayerShootResult playerShootResult) {
+    public void listenOnJudgePlayerShootResultQueue(PlayerShootResult playerShootResult) {
         logger.logInfo(JudgeQueueListener.class, "Received message" + playerShootResult);
 
         ShootResult shootResult = playerShootResult.getShootResult();
 
-        RoundStatus roundStatus = RoundStatus.GAME_END;
-
-        if (shootResult.equals(ShootResult.HIT)) {
-            roundStatus = RoundStatus.SAME_PLAYER;
-        }
-        else if (shootResult.equals(ShootResult.MISS)) {
-            roundStatus = RoundStatus.NEXT_PLAYER;
+        RoundStatus roundStatus;
+        switch (shootResult) {
```

Comment:
playerShootResult.getShootResult()

Reply:
done according to articles like:
https://blogs.msdn.microsoft.com/cellfish/2009/08/19/object-calisthenics-rule-4-use-only-one-dot-per-line/